SU2_CFD/src/output/CFlowIncOutput.cpp

```diff
@@ -111,12 +112,12 @@ void CFlowIncOutput::SetHistoryOutputFields(CConfig *config){
   if (heat || weakly_coupled_heat) AddHistoryOutput("RMS_TEMPERATURE", "rms[T]", ScreenOutputFormat::FIXED, "RMS_RES", "Root-mean square residual of the temperature.", HistoryFieldType::RESIDUAL);
 
   AddHistoryOutputFields_TurbRMS_RES(config);
+  AddHistoryOutputFields_SpeciesRMS_RES(config);
```

Comment:
Maybe it is better to make just one `AddHistoryOutputFields_ScalarRMS_RES` instead of one for Turb and one for Species

Reply:
did that in 8d87b35